x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1356,39 +1415,108 @@ public static void registerSettings(List<Setting<?>> settings) {
          * a singleton list of the empty string ([""]).
          */
         EventFilterPolicy(String name, Predicate<String> ignorePrincipalsPredicate, Predicate<String> ignoreRealmsPredicate,
-                Predicate<String> ignoreRolesPredicate, Predicate<String> ignoreIndicesPredicate) {
+                Predicate<String> ignoreRolesPredicate, Predicate<String> ignoreIndicesPredicate,
+                Predicate<String> ignoreIndexPrivilegesPredicate, Predicate<String> ignoreClusterPrivilegesPredicate) {
             this.name = name;
             // "null" values are "unexpected" and should not match any ignore policy
             this.ignorePrincipalsPredicate = ignorePrincipalsPredicate;
             this.ignoreRealmsPredicate = ignoreRealmsPredicate;
             this.ignoreRolesPredicate = ignoreRolesPredicate;
             this.ignoreIndicesPredicate = ignoreIndicesPredicate;
+            this.ignoreIndexPrivilegesPredicate = ignoreIndexPrivilegesPredicate;
+            this.ignoreClusterPrivilegesPredicate = ignoreClusterPrivilegesPredicate;
+
+            if (ignoreIndexPrivilegesPredicate.test("") == false && ignoreClusterPrivilegesPredicate.test("") == false) {
+                final String message = String.format(
+                    Locale.ROOT,
+                    "Both Index and Cluster privilege ignore filters are set for policy [%s]. " +
+                        "Please update your configuration settings and remove one of these filters to avoid ambiguity.", this.name);
+                throw new IllegalArgumentException(message);
+            }
+
+            if (ignoreClusterPrivilegesPredicate.test("") == false) {
+                ClusterPermission.Builder builder = ClusterPermission.builder();
+                String[] clusterPrivileges = ignoreClusterPrivilegesPredicate.toString().split("\\|");
+                for (String privilege : clusterPrivileges) {
+                    builder = ClusterPrivilegeResolver.resolve(privilege).buildPermission(builder);
+                }
+                this.clusterPermission = builder.build();
+            } else {
+                this.clusterPermission = ClusterPermission.NONE;
+            }
+
+            if (ignoreIndexPrivilegesPredicate.test("") == false) {
+                List<IndicesPermission.Group> groups = new ArrayList<>();
+                Set<String> set = new HashSet<>(Arrays.stream(ignoreIndexPrivilegesPredicate.toString().split("\\|"))
+                    .collect(Collectors.toSet()));
+                groups.add(new IndicesPermission.Group(IndexPrivilege.get(set), FieldPermissions.DEFAULT, null, false, "*"));
```

Comment:
You don't want an `IndicesPermission#Group` here. It does a lot more stuff than the checks we're after in this context (eg, data streams, backing indices, auto mapping updates).
We discussed that we don't want (can't really implement it) ignore policies for indices and index actions to work exactly like role permissions.

Instead, take the automaton from `IndexPrivilege`.